README.md

```diff
@@ -16,72 +16,72 @@ Backed by Y Combinator
+Fern supports the generation of development resources such as Postman collections, and OpenAPI specifications, making integration into existing development workflows effortless.
 
-Read [Fern vs. OpenAPI](https://buildwithfern.com/docs/comparison-with-openapi)
-for an in-depth comparison. **TL;DR: we differ from OpenAPI in these areas:**
+## Fern and OpenAPI
 
-- [Simplicity](https://www.buildwithfern.com/docs/comparison-with-openapi#simplicity)
-- [New features in specification](https://www.buildwithfern.com/docs/comparison-with-openapi#new-features-in-specification)
-- [Quality of code generation](https://www.buildwithfern.com/docs/comparison-with-openapi#quality-of-code-generation)
-- [Focus on server-side API development](https://www.buildwithfern.com/docs/comparison-with-openapi#focus-on-server-side-api-development)
-- [Change management](https://www.buildwithfern.com/docs/comparison-with-openapi#change-management)
-- [Cloud-based code generation and publishing](https://www.buildwithfern.com/docs/comparison-with-openapi#cloud-based-code-generation-and-publishing)
+We have designed Fern to complement existing OpenAPI toolchains and workflows, rather than replace them. We believe OpenAPI does a good job at as a declarative standard for defining APIs, but at times it is too verbose and complex, which harms the quality of automatically generated code.
 
-# Plant Store example
+The goal of Fern is to be **complimentary to OpenAPI**, not to replace it. As such, you are never locked in to Fern, its specifications, or its tools.
 
-Plant Store is full example of an API defined in Fern.
+If you want to read more about where Fern and OpenAPI differ, read [﻿Fern vs. OpenAPI](https://buildwithfern.com/docs/comparison-with-openapi) for an in-depth comparison.
 
-- [API Definition](https://github.com/fern-api/plantstore-api)
-- [Generated TypeScript SDK](https://github.com/fern-api/plantstore-node)
-- [Generated Java SDK](https://github.com/fern-api/plantstore-java)
-- [Generated Postman Collection](https://github.com/fern-api/plantstore-postman)
-- [Generated OpenAPI spec](https://github.com/fern-api/plantstore-openapi)
+**TL;DR: we differ from OpenAPI in these areas:**
 
-# Get started
+-   [﻿New features in specification](https://www.buildwithfern.com/docs/comparison-with-openapi#new-features-in-specification)
+    
+-   [﻿Quality of code generation](https://www.buildwithfern.com/docs/comparison-with-openapi#quality-of-code-generation)
+    
+-   [﻿Focus on server-side API development](https://www.buildwithfern.com/docs/comparison-with-openapi#focus-on-server-side-api-development)
+    
+-   [﻿Change management](https://www.buildwithfern.com/docs/comparison-with-openapi#change-management)
+    
+-   [﻿Cloud-based code generation and publishing](https://www.buildwithfern.com/docs/comparison-with-openapi#cloud-based-code-generation-and-publishing)
+    
+# Getting Started
```

Comment:
going back and forth between "if you have an openapi" and "if you are starting from scratch" is hard for me to process as a reader.

think it makes sense to have two getting started sections here?

Reply:
Agreed, this was something I was struggling with when writing. Moving `--openapi`  into a separate OpenAPI section.